ios-11-4-dep-9-4-arm64-hid-sections-lto.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2017-2018, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_IOS_11_4_DEP_9_4_ARM64_HID_SECTIONS_LTO_CMAKE_)
+  return()
+else()
+  set(POLLY_IOS_11_4_DEP_9_4_ARM64_HID_SECTIONS_LTO_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_clear_environment_variables.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+set(IOS_SDK_VERSION 11.4)
+set(IOS_DEPLOYMENT_SDK_VERSION 9.4)
+
+set(POLLY_XCODE_COMPILER "clang")
+polly_init(
+    "iOS ${IOS_SDK_VERSION} / Deployment ${IOS_DEPLOYMENT_SDK_VERSION} / Universal (arm64) / \
+${POLLY_XCODE_COMPILER} / \
+c++11 support"
+    "Xcode"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include(polly_fatal_error)
+
+# Fix try_compile
+include(polly_ios_bundle_identifier)
+
+set(CMAKE_MACOSX_BUNDLE YES)
+set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "iPhone Developer")
+
+set(IPHONEOS_ARCHS arm64)
+set(IPHONESIMULATOR_ARCHS "")
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/xcode.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/os/iphone.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx11.cmake")
```

Comment:
Since iOS 11.0 default standard is C++14, if we need C++11 toolchain name should ends with `-cxx11`